satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -32,6 +32,7 @@ model pending_audits (
 	field share_size          int64
 	field expected_share_hash blob
 	field reverify_count      int64 ( updatable )
+	field path                text
```

Comment:
How are we going to migrate the existing entry in the `containment` table?

Reply:
we're not.  we're just going to not care.  it's extraordinarily freeing.

Reply:
we will delete them all

Reply:
nevermind, we will leave the paths blank and our logic will treat things without paths the way it used to.